Configurations/unix-Makefile.tmpl

```diff
@@ -679,9 +679,12 @@ tags TAGS: FORCE
 
 # Release targets (note: only available on Unix) #####################
 
+# If your tar command doesn't support --owner and --group, make sure to
+# use one that does, for example GNU tar
 TAR_COMMAND=$(TAR) $(TARFLAGS) --owner 0 --group 0 -cvf -
```

Comment:
Nah.  Set the whole `TAR_COMMAND` then.  See [`.travis-create-release.sh`](https://github.com/openssl/openssl/blob/master/.travis-create-release.sh), which does exactly that for Mac OS X.